### ISSUE TITLE
Evaluate the Model

### ISSUE THREAD

**Issue by Nishant001-jyo** [Nishant001-jyo/Nishant001-jyo#7]
`from sklearn.metrics import accuracy_score, classification_report, confusion_matrix

y_pred = model.predict(X_test)
print("Accuracy:", accuracy_score(y_test, y_pred))
print(classification_report(y_test, y_pred, target_names=iris.target_names))
print(confusion_matrix(y_test, y_pred))`
